py-polars/polars/dataframe/frame.py

```diff
@@ -7137,6 +7137,29 @@ def pivot(
         │ b    ┆ 0.964028 ┆ 0.999954 │
         └──────┴──────────┴──────────┘
 
+        Set the index to None to output a single row.
+        >>> df = pl.DataFrame(
```

Comment:
are you sure this renders properly? i think you need an extra blank line

Reply:
Ah yes I didn't see that. Fixed now.